setup.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
We recently switched to using `pyproject.toml` instead of the `setup.json`. Please add your changes to that file instead.

Reply:
Have now deleted `setup.json` and adding the relevant gpaw keywords to `pyproject.toml`